.github/CODEOWNERS

```diff
@@ -38,12 +43,13 @@ android/res/values-zh-rTW/ @organicmaps/translations-zh-Hant
 iphone/Maps/LocalizedStrings/ar.lproj/ @organicmaps/translations-ar
 iphone/Maps/LocalizedStrings/be.lproj/ @organicmaps/translations-be
 iphone/Maps/LocalizedStrings/bg.lproj/ @organicmaps/translations-bg
+iphone/Maps/LocalizedStrings/ca.lproj/ @organicmaps/translations-ca
 iphone/Maps/LocalizedStrings/cs.lproj/ @organicmaps/translations-cs
 iphone/Maps/LocalizedStrings/da.lproj/ @organicmaps/translations-da
 iphone/Maps/LocalizedStrings/de.lproj/ @organicmaps/translations-de
 iphone/Maps/LocalizedStrings/el.lproj/ @organicmaps/translations-el
 iphone/Maps/LocalizedStrings/es.lproj/ @organicmaps/translations-es
-iphone/Maps/LocalizedStrings/es-MX.lproj/ @organicmaps/translations-es
+iphone/Maps/LocalizedStrings/es-MX.lproj/ @organicmaps/translations-es-MX
```

Comment:
Do we have enough people to have separate teams for es and es-MX? I have doubts about that.

Reply:
We WILL have them sooner or later. It is easier to exactly match our existing languages, and then add contributors when we find them (for example, when someone contributes changes to these languages, he/she can be asked and added into that team).

Reply:
Are `es` and `es-MX` really require two completely separate groups of people to review? I have doubts.

Reply:
I've fixed master. Are you sure that we actually need to split `es` and `es-MX`? I am not a big expert in Español, but I thought that Mexican variation is not significantly different. It is not like Simplified vs. Traditional Chinese.

Reply:
There are many different words in EU and Latin America.

It is the same as GB and US case.

Reply:
Let's get to the point when we have 2+ folks from both side and when split the team. Currently this is a premature optimization nobody asked us to do.

Reply:
It's inevitable and only a question of time. Now it will take 5 min of your time. Later it will take 25 min.